nengo_gui/components/netgraph.py

```diff
@@ -197,6 +207,7 @@ def _reload(self, code=None):
                 # this is a Component that was previously removed,
                 #  but is still in the config file, or it has to be
                 #  rebuilt, so let's recover it
+                # This is also how new components are created
```

Comment:
I'm not sure what this comment means.  It's not how new components are normally created, since that just involves right-clicking on netgraph items and choosing them from the list, which doesn't involve reloading at all.